packages/drawer/src/ux-drawer.html

```diff
@@ -1,9 +1,16 @@
-<template class="ux-drawer ux-drawer--${computedType} ux-drawer--${viewportType} ux-drawer--${position} ${animate ? 'ux-drawer--animate':''} ${showed ? 'ux-drawer--opened':''}">
+<template 
+  class="ux-drawer ux-drawer--${computedType} ux-drawer--${viewportType} ux-drawer--${position} ${animate ? 'ux-drawer--animate':''} ${showed ? 'ux-drawer--opened':''}">
   <require from="./ux-drawer.css"></require>
   <div class="ux-drawer__overlay" ref="overlayElement">
   </div>
-  <div class="ux-drawer__content-wrapper" ref="contentWrapperElement" click.delegate="overlayClick($event)">
-    <div class="ux-drawer__content" ref="contentElement" click.delegate="stop($event)">
+  <div 
+    role="${role || 'dialog'}"
+    aria-labelledby="ariaLabelledBy"
+    aria-describedby="ariaDescribedBy"
```

Comment:
@EisenbergEffect

For the `role` I've added a bindable that defaults to `dialog` (user can set it to `alertdialog` which is also valid for some cases.

Regarding `arial-labelledby` and `aria-describedby` the are content specific. Therefore I leave them here as bindable values. Users can set them according to their usage.

Once the core features of this drawer/dialog component are set I would like to provide a few utility methods with a default template to quickly display dialogs such as alerts, prompt and confirm. I would also like to provide an empty template for dialogs and users who like to have the template ready. I'm planning on settings these templates with `ux-card` as I do now in the demo.

Why I say all that is that for these cases I can set the correct aria properties directly. But for all user custom implementation then I can't do it for them.

Reply:
That makes sense and this all looks good. Thanks!